platform/backend/core/src/integrationTest/kotlin/io/hamal/core/req/handler/snippet/CreateSnippetHandlerTest.kt

```diff
@@ -0,0 +1,45 @@
+package io.hamal.core.req.handler.snippet
+
+import io.hamal.core.req.handler.BaseReqHandlerTest
+import io.hamal.lib.domain.ReqId
+import io.hamal.lib.domain._enum.ReqStatus
+import io.hamal.lib.domain.vo.*
+import io.hamal.lib.kua.type.MapType
+import io.hamal.lib.kua.type.StringType
+import io.hamal.repository.api.submitted_req.SubmittedCreateSnippetReq
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+
+internal class CreateSnippetHandlerTest : BaseReqHandlerTest() {
+
+    @Autowired
+    private lateinit var testInstance: CreateSnippetHandler
```

Comment:
variables at the end of the file

general structure

tests
helper functions 
variables

Reason being, you start reading the file most of the time top to bottom. You usually care about tests and not the variable defiinitions